libc/src/__support/OSUtil/linux/fcntl.cpp

```diff
@@ -19,6 +19,14 @@
 #include <stdarg.h>
 #include <sys/syscall.h> // For syscall numbers.
 
+#if SYS_fcntl
+constexpr auto FCNTL_SYSCALL_ID = SYS_fcntl;
```

Comment:
Can you put `FCNTL_SYSCALL_ID` into an anonymous namespace to keep it local.

Reply:
I can actually make this a local variable, thanks for the suggestion